### PR TITLE
Update _index.md  Added some more orienting text to Resources

### PR DESCRIPTION
**Pull request by memartone**
Feel free to modify;  just trying to fill in as much as possible before the EAB

### CODE REVIEW

File: content/resources/_index.md [hunk 1]
```diff
@@ -5,5 +5,8 @@ type: docs
 
 TODO: [GH 74](https://github.com/ReproNim/repronim.org/issues/74)
 
-This section is dedicated to **how** to make use of the ReproNim ecosystem.
-Many of the materials here assume you are already familiar with [why neuroimaging should be reproducible](/about/why/).
+This section is dedicated to **how** to make use of the ReproNim ecosystem, comprising tools, training materials and tutorials on how to use ReproNim resources to improve the reproducibility of your neuroimaging studies.  
+
+If you are new to the concept of ReproNim's mission to increase reproducible practices in neuroimaging, we suggest you familiarize yourself with [why neuroimaging should be reproducible](/about/why/) and [ReproNim's Principles of Reproducible Neuroimaging](https://repronim.netlify.app/about/in-practice/).
```
Comment (asmacdo) [inline content/resources/_index.md:10]:
```suggestion
If you are new to the concept of ReproNim's mission to increase reproducible practices in neuroimaging, we suggest you familiarize yourself with [why neuroimaging should be reproducible](/about/why/) and [ReproNim's Principles of Reproducible Neuroimaging](/about/in-practice/). 
```

File: content/resources/_index.md [hunk 2]
```diff
@@ -5,5 +5,8 @@ type: docs
 
 TODO: [GH 74](https://github.com/ReproNim/repronim.org/issues/74)
 
-This section is dedicated to **how** to make use of the ReproNim ecosystem.
-Many of the materials here assume you are already familiar with [why neuroimaging should be reproducible](/about/why/).
+This section is dedicated to **how** to make use of the ReproNim ecosystem, comprising tools, training materials and tutorials on how to use ReproNim resources to improve the reproducibility of your neuroimaging studies.  
+
+If you are new to the concept of ReproNim's mission to increase reproducible practices in neuroimaging, we suggest you familiarize yourself with [why neuroimaging should be reproducible](/about/why/) and [ReproNim's Principles of Reproducible Neuroimaging](https://repronim.netlify.app/about/in-practice/). 
+
+If you are unfamiliar with ReproNim and its approach, we suggest you visit [Getting started with ReproNim](https://repronim.netlify.app/resources/getting-started/) for a general orientation to the website, along with a set of user stories that illustrate how ReproNim can address data and software management issues you may encounter in your current neuroimaging worklow
```
Comment (asmacdo) [inline content/resources/_index.md:12]:
```suggestion
If you are unfamiliar with ReproNim and its approach, we suggest you visit [Getting started with ReproNim](/resources/getting-started/) for a general orientation to the website, along with a set of user stories that illustrate how ReproNim can address data and software management issues you may encounter in your current neuroimaging worklow
```

**Review by asmacdo — APPROVED** [review]
Looks good to me, I replaced 2 absolute links with relative links, otherwise looks good to me!

**Comment by chaselgrove** [comment]
Closes #74.

**Pull request merged**